From eb46e5a9f124d7da98a225d248f6b4927768bf5f Mon Sep 17 00:00:00 2001
From: bobcanthelpyou <bobcanthelpyou@users.noreply.github.com>
Date: Sat, 23 Mar 2019 18:37:49 +0100
Subject: [PATCH] docs: add new <%| tag for escaped expressions (#1683)

the tag was added by dd23a805c2a1edf7f1df293dd03a542f3b2b3d9e

(cherry picked from commit cae79eb07426b2a9f87b4d0a4d5a07ecebfd6558)
---
 docs/dev/web/view.rst | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/docs/dev/web/view.rst b/docs/dev/web/view.rst
index d4f63b60..9f02df2d 100644
--- a/docs/dev/web/view.rst
+++ b/docs/dev/web/view.rst
@@ -11,7 +11,9 @@ Views are partial HTML pages, with additional template tags that allow
 to embed Lua code and translation strings. The following tags are defined:
 
   - ``<%`` ... ``%>`` evaluates the enclosed Lua expression.
-  - ``<%=`` ... ``%>`` evaluates the enclosed Lua expression and prints its value.
+  - ``<%|`` ... ``%>`` evaluates the enclosed Lua expression and prints its value.
+  - ``<%=`` ... ``%>`` evaluates the enclosed Lua expression and prints its value
+    *without escaping HTML entities*. This is useful when the value contains HTML code.
   - ``<%+`` ... ``%>`` includes another template.
   - ``<%:`` ... ``%>`` translates the enclosed string using the loaded i18n catalog.
   - ``<%_`` ... ``%>`` translates the enclosed string *without escaping HTML entities*
-- 
2.17.1

